src/Tools/Extensions.ApiDescription.Client/src/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -25,7 +25,6 @@
       @(OpenApiProjectReference) items.
     -->
     <ProjectReference Include="@(OpenApiProjectReference)" Exclude="@(ProjectReference)">
-      <GlobalPropertiesToRemove>TargetFramework</GlobalPropertiesToRemove>
       <Private>false</Private>
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
```

Comment:
@rainersigwald should this also set `%(ShouldUnsetParentConfigurationAndPlatform)`?

Reply:
Is it expected/allowed to have a different API surface between configurations/platforms? Like say a debug-only endpoint or something?

I would generally _not_ set that; it makes sense to let the user's solution configuration for those things flow here, IMO.

Reply:
Thanks!